### PR TITLE
Mm add tests new item and right arrow

### PR DESCRIPTION
**Pull request by MichaelMicrof**
https://trello.com/c/U5qH1WTr/27-menu-selector-right-arrow-push-tc-005
https://trello.com/c/umdGxldW/28-dashboard-new-item-clickable-tc-010

### CODE REVIEW

File: tests/locators_dashboard_page.py [hunk 1]
```diff
@@ -65,3 +74,7 @@ class URLLocators:
     URL_NEW_VIEW = 'http://localhost:8080/newView'
     URL_FOOTER_REST_API = 'http://localhost:8080/api/'
     URL_FOOTER_VERSION = 'https://www.jenkins.io/'
+
+class Titles:
+    TITLE_DASHBOARD_PAGE = 'Dashboard [Jenkins]'
+    TITLE_NEW_ITEM = 'New Item [Jenkins]'
```
Comment (Python2-QAForEveryone) [inline tests/locators_dashboard_page.py:80]: I assume this Title belongs to different page. That means it shouldn't be here.

File: tests/test_dashboard_page.py [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 from pages.BasePage import BasePage
 from pages.DashboardPage import DashboardPage
-from tests.locators_dashboard_page import DashboardPageLocators, FooterLocators, BuildLocators, AddDescriptionLocators
+from config.TestData import TestData as TD
+from tests.locators_dashboard_page import DashboardPageLocators, FooterLocators, BuildLocators, AddDescriptionLocators, \
+    Titles
 
 
 class TestHomePage:
```
Comment (Python2-QAForEveryone) [inline tests/test_dashboard_page.py:10]: There is no such page as "HomePage"
i think it should be change to "DashboardPage"

File: tests/test_dashboard_page.py [hunk 3]
```diff
@@ -25,16 +27,32 @@ def test_footer_all_elements_visible_and_clickable_tc_002(self, locator):
         assert driver.is_visible(locator)
         assert driver.is_clickable(locator)
 
-    @pytest.mark.parametrize('locator', DashboardPageLocators.locators_dashboard_all)
+    def test_menu_selector_can_be_push_tc_005(self):
```
Comment (Python2-QAForEveryone) [inline tests/test_dashboard_page.py:30]: Could you please clarify what you mean under "can_be_push"?

File: tests/test_dashboard_page.py [hunk 4]
```diff
@@ -25,16 +27,32 @@ def test_footer_all_elements_visible_and_clickable_tc_002(self, locator):
...
+                             ids=DashboardPageLocators.ids_dashboard_text_field)
     def test_dashboard_all_element_is_clickable_tc_009(self, locator):
         driver = DashboardPage(self.driver)
         assert driver.is_clickable(locator)
 
+    def test_dashboard_new_item_clickable_tc_010(self):
+        driver = DashboardPage(self.driver)
+        driver.click(DashboardPageLocators.TEXT_NEW_ITEM)
+        assert driver.get_title() == Titles.TITLE_NEW_ITEM
+        driver.go_to_page(TD.BASE_URL)
```
Comment (Python2-QAForEveryone) [inline tests/test_dashboard_page.py:54]: It looks like this test is verifying title of a different page, i mean not DashboardPAge.If it is, it belongs to a different test page.

**Review by Python2-QAForEveryone — CHANGES_REQUESTED** [review]

File: tests/locators_dashboard_page.py (hunk 1 above)
Comment (MichaelMicrof) [reply tests/locators_dashboard_page.py:80]: ok, I'll change it

File: tests/test_dashboard_page.py [hunk 5]
```diff
@@ -2,7 +2,9 @@
 
 from pages.BasePage import BasePage
 from pages.DashboardPage import DashboardPage
-from tests.locators_dashboard_page import DashboardPageLocators, FooterLocators, BuildLocators, AddDescriptionLocators
+from config.TestData import TestData as TD
+from tests.locators_dashboard_page import DashboardPageLocators, FooterLocators, BuildLocators, AddDescriptionLocators, \
+    Titles
```
[the remark this answers is not in the document]
Comment (MichaelMicrof) [reply tests/test_dashboard_page.py:7]: It's automatics, I'll change it

File: tests/test_dashboard_page.py (hunk 2 above)
Comment (MichaelMicrof) [reply tests/test_dashboard_page.py:10]: ok

File: tests/test_dashboard_page.py (hunk 4 above)
Comment (MichaelMicrof) [reply tests/test_dashboard_page.py:54]: I have a problem with driver.current_url - I need help to solve it

File: tests/test_dashboard_page.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (MichaelMicrof) [reply tests/test_dashboard_page.py:7]: done

File: tests/test_dashboard_page.py (hunk 3 above)
Comment (MichaelMicrof) [reply tests/test_dashboard_page.py:30]: done

File: tests/test_dashboard_page.py (hunk 4 above)
Comment (MichaelMicrof) [reply tests/test_dashboard_page.py:54]: now added skip this test

File: tests/locators_dashboard_page.py (hunk 1 above)
Comment (MichaelMicrof) [reply tests/locators_dashboard_page.py:82]: now this test is skipping.

**Review by Python2-QAForEveryone — APPROVED** [review]
Good job

**Pull request merged**